pkgs/servers/varnish/dynamic.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, fetchFromGitHub, automake, autoconf, libtool, pkgconfig, varnish, python, docutils }:
+
+stdenv.mkDerivation rec {
+  version = "0.3";
+  name = "${varnish.name}-dynamic-${version}";
+
+  src = fetchFromGitHub {
+    owner = "nigoroll";
+    repo = "libvmod-dynamic";
+    rev = "475be183fddbd727c3d2523f0518effa9aa881f8"; # 5.2 branch for Varnish-5.2 https://github.com/nigoroll/libvmod-dynamic/commits/5.2
+    sha256 = "12a42lbv0vf6fn3qnvngw893kmbd006f8pgab4ir7irc8855xjgf";
+  };
+
+  nativeBuildInputs = [ pkgconfig docutils automake autoconf libtool ];
+  buildInputs = [ varnish python ];
+  preConfigure = ''
+    substituteInPlace autogen.sh --replace 'pkg-config --variable=datarootdir varnishapi' 'echo ${varnish.dev}/share'
+    ./autogen.sh
```

Comment:
do we actually need to run their `autogen.sh`? It seems not to do much more then `autoreconfHook`.

Reply:
yes, changed to ```autoreconfHook```